config/config.go

```diff
@@ -282,6 +284,9 @@ func saveConfig(cfg *Config) *Config {
 			core.AutoComplete = true
 			core.Output = JSON
 		}
+		if conf.Section(ini.DEFAULT_SECTION).HasKey("postrequest") {
```

Comment:
```suggestion
		if !conf.Section(ini.DEFAULT_SECTION).HasKey("postrequest") {
			core.PostRequest = false		
```

Reply:
@kevin-lii , @sureshanaparti sugested `**_!_**conf.Section(ini.DEFAITL_SECTION).HasKey(“postrequest”)`, you have without the negation. Are you sure this is what you intend to do?

Reply:
`if !conf.Section(ini.DEFAULT_SECTION).HasKey("postrequest") {` - cond. should be this one, right @kevin-lii ?